README.md

```diff
@@ -0,0 +1,52 @@
+# Tech Docs Template
+
+The Tech Docs Template is a static site generator to build technical documentation with a GOV.UK style for GaaP (Government as a Platform) products.
+
+## Getting started
+
+To preview or build the website, we need to use the terminal.
+
+Install Ruby with Rubygems, perferably with a [Ruby version manager](https://www.ruby-lang.org/en/documentation/installation/#managers), and the [Bundler gem](http://bundler.io/).
+
+In the application folder type the following to install the required gems:
+
+```
+$ bundle install
+```
+
+## Preview
+
+We can run a server that let's us see changes as we make them.
```

Comment:
It's probably worth mentioning the live reload functionality here, and fleshing this out a bit.

> "Whilst writing documentation you can run middleman server to preview how the published version will look in your browser. Whenever you save your changes the preview in your browser will automatically refresh."

Might also be worth making it clear that this is a local preview…

> "Note that the preview is only available on your computer – others won't be able to access it even if you send them a link."